src/nvim/buffer.c

```diff
@@ -4610,6 +4618,25 @@ chk_modeline (
 }
 
 /*
+ * checks the modeline is not longer
```

Comment:
Do this only for newly added comments, and lines immediately surrounding any changes introduced. Run clint to get the list of changes required.

The codebase is littered with `/* ...*/` style comments and updating all of them at once will make git-blaming very difficult, hence this policy was chosen to update things incrementally.